docs/getting-started.md

```diff
@@ -1,9 +1,21 @@
 ---
 id: getting-started
-title: Getting Started (vnext)
+title: Getting Started
 ---
 
-This is a summary of setup steps needed to install and work with React Native for Windows (vnext). See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for React Native details and see [Getting Started Guide - current](https://github.com/microsoft/react-native-windows/blob/master/current/docs/GettingStarted.md) for working with the `current` React Native for Windows implementation.
+This is a summary of all the steps needed to get setup to install a React Native for Windows app.

```

Comment:
Given how condensed this is, perhaps we want to add the consuming-react-native-windows.md in place in this doc instead of one layer of indirection. The other part about building the whole repo can remain separate - that is only for advanced contributors.

Reply:
I broke it up into getting started - which has the steps. And "System Requirements" which has the old getting started information.